Update editions-and-offerings.rst (#5681)
mattermost/docs: source/onboard/mattermost-limits.rst

Workspace limits
================

> There may be limits applied to your Mattermost Cloud workspace, based on the type of subscription plan you have. These limits do not apply to self-hosted deployments.

Reviewer: Propose adding a paragraph/sentence somewhere that there are additional features that paid subscriptions unlocked, with a complete list available in https://mattermost.com/pricing/

(Want to avoid confusion where someone thinks apart from these limits, all other features are available in the Cloud Starter plan)

Author: Good call. I'll add it.

These limits are applied across all users in your team. We'll let you know when you're halfway from reaching a limit, as well as when you're closer to it. 

> When limits have been reached, data is archived. This data is not deleted, and can be accessed when you upgrade to a plan with higher limits. If you choose to downgrade your plan, for example from Mattermost Professional to Mattermost Starter, the limits will be applied to your existing data.

Reviewer: Not a blocker to merge, but there may be a question that when you downgrade to Starter (either from trial or paid plan), what happens? Files/messages/cards is self-explanatory as you'll just see the most recent information. 

How about teams, integrations or views -- which ones will be retained upon downgrade?

Author: I'm following this up with John and it'll be in the next iteration of this content. :)

Mattermost Starter limits
-------------------------

**Platform limits**

- 1 team.
- 10 GB file storage across the platform, with 100MB upload limit.
- 5 apps/plugins enabled.

**Channels limits**

- 10,000 message history limit.

**Boards limits**

- 5 views per board.
- Only the 500 most recently updated cards on the server are displayed.

Mattermost Professional limits
------------------------------

**Platform limits**

- 250GB file storage limit.

Frequently Asked Questions
--------------------------

Can I upgrade my plan to get a copy of my archived data?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Currently you don't have to upgrade your plan to get your archived data. You can follow the steps provided `in this article <https://docs.mattermost.com/manage/cloud-data-export.html>`_ to export your data.

Does Mattermost delete data?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> When file limits are reached, your data is archived. You can export your data. However, if you don't log into, or use, your workspace for an extended period of time, your data may be deleted. We'll let you know via about this ahead of time though, so you can take action.

Reviewer: ```suggestion
When file and message limits are reached, your data is archived. While you can export your data anytime, if you don't log into, or use, your workspace for an extended period of time, your data may be deleted. We'll let you know ahead of time, so you can take action to reactivate your workspace.
```
